packages/task/src/node/process/process-task-runner.ts

```diff
@@ -40,6 +40,19 @@ import { ProcessTaskError, CommandOptions } from '../../common/process/task-prot
 import * as fs from 'fs';
 import { ShellProcess } from '@theia/terminal/lib/node/shell-process';
 
+interface OsSpecificCommand {
+    command: string,
+    args: Array<string | ShellQuotedString> | undefined,
```

Comment:
`args?:`

Reply:
This one is intentionally this way; otherwise a type error occurs.